ci-operator/jobs/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main-presubmits.yaml

```diff
@@ -159,4 +158,4 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )test-e2e-grc-framework,?($|\s.*)
+    trigger: (?m)^/test( | .* )(test-e2e-grc-framework|remaining-required),?($|\s.*)
```

Comment:
What is the `remaining-required` for?

Reply:
Since it's a "secondary" job intended to be triggered manually, it can be triggered with `/test remaining-required` (which would also apply to other secondary jobs if we had any).

ref: https://docs.ci.openshift.org/docs/how-tos/creating-a-pipeline/#manual-pipelines